qiskit/algorithms/amplitude_amplifiers/amplitude_amplifier.py

```diff
@@ -100,3 +102,23 @@ def oracle_evaluation(self, value: bool) -> None:
             value: A new value for the classical oracle evaluation.
         """
         self._oracle_evaluation = value
+
+    @property
+    def circuit_results(self) -> Optional[Union[np.ndarray, Dict[str, int]]]:
```

Comment:
This is just the result of the final (may be the only) circuit that is run. I.e. if we run a bunch of circuits as part of an interative solution it is not the results of that set of circuits (since its not a List here and if it were I would expect that) just the result of the last circuit. Maybe that could be made clearer in the docstring and the name perhaps should be `circuit_result` without the `s` - or perhaps a different name like `final_measurement` or something  - maybe someone else has some better suggestion especially since this is in the base result class here.

Reply:
Thank you for your comment. Ya, I changed it to store the result of all the circuits that are executed as a list. I forgot to change the typehint. I will fix it quickly.